native/navigation/default-state.js

```diff
@@ -38,9 +39,12 @@ const defaultNavigationState: StaleNavigationState = {
                 {
                   name: ChatRouteName,
                   state: {
-                    type: 'stack',
+                    type: 'tab',
                     index: 0,
-                    routes: [{ name: ChatThreadListRouteName }],
+                    routes: [
+                      { name: HomeChatThreadListRouteName },
```

Comment:
Why did you replace the stack here? `ChatThreadListRouteName` isn't gone, it just has a child now